File: client/src/app/_dtos/admin/visaBriefDto.ts
export interface IVisaBriefDto
{
    id: number;
    visaItemId: number;
    customerKnownAs: string;
    customerId: number;
    visaNo: string;
    visaConsulate: string;
    visaDateH: string;
    visaDateG: Date;
    visaCategoryEnglish: string;
    visaSponsorName: string;
    visaQuantity: number;
    visaConsumed: number;
    visaBalance: number;
    depItemId: number;
    visaCanceled: boolean;
}

File: api/DTOs/Admin/VisaBriefDto.cs
namespace api.DTOs.Admin
{
    public class VisaBriefDto
    {
        public int Id { get; set; }
        public int VisaItemId { get; set; }
        public string CustomerKnownAs { get; set; }
        public int CustomerId { get; set; }
        public string VisaNo { get; set; }
        public bool VisaCanceled { get; set; }
        public string VisaDateH { get; set; }
        public DateTime VisaDateG { get; set; }
        public string VisaConsulate { get; set; }
        public string VisaCategoryEnglish { get; set; }
        public string VisaSponsorName { get; set; }
        public int VisaQuantity { get; set; }
        public int VisaConsumed { get; set; }
        public int VisaBalance { get; set; }
        public int DepItemId { get; set; }
    }
}
